src/Symfony/Component/VarExporter/Tests/Fixtures/LazyGhost/RegularClass.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarExporter\Tests\Fixtures\LazyGhost;
+
+
+if (\PHP_VERSION_ID < 80300) {
```

Comment:
why only for PHP 8.2 and older ?

Reply:
I planned to remove that class in SF8 but that's not needed anymore